[WIP - do not merge] Content section for writing guidelines in EUI (#7710)
elastic/eui: website/docs/03_content/adapt-tone.mdx

---
slug: tone
title: Adapt your tone
description: Adapt the way you write to each situation users can face. 
---

Adapt the way you write to each situation users can face. 

 Our products and solutions are very technical, data-oriented, and they're used globally by people from different cultures, abilities, and personalities. 
 These considerations, amongst others, limit the spectrum of tones we use to some extent, but not entirely. 
 
 During the users' journey with Elastic, there are times for excitement and discovery, but also times for help, support, and empathy.

 ## Stimulating

> **Attributes**

Author: Yeah, don't hesitate to rearrange this. As of now this is simply flattening the structure of the page as it was on docs.elastic.dev, but without any specific docsmobile components, so it's less digestible.

    - Take time to explain what's so stimulating about what you're writing about.
    - Use visuals to support your text.

**Cases**

    Although it can be very visible because of the way it's written and the UI elements it shows in, the **stimulating** tone is used **quite rarely** overall. 
    
    We use it when we want to show new things to users, especially in:

    - Product tours
    - New feature announcements

## Friendly

**Attributes**

    All's fine, we'll help you find your way. Be:

    - Calm
    - Positive
    - Confident

    When using this tone, you may:
    - Use more casual terminology.
    - Explain what's missing or what's happening.
    - Provide actions or clearly state when there's no action needed.

**Cases**

    The **friendly** tone is used **sparingly** overall and usually blends in the UI at different moments of the users' journey. 
    
    We use it when orienting users in doing their next action or learning different things they could do, especially in:

    **UI**
    - Empty states (no item created or no matching results)
    - Loading and waiting states
    - Tips
    - Success messages
    - Welcome emails (signup & trial)

>     **Documentation**

Author: We can remove it, or we can keep it as a reference point for those who read

    - Quick starts
    - Tutorials

## Informational

**Attributes**

>     E=MC². Be:

Reviewer: I think the formula here just represents to be direct, straight to the point, and neutral. It's the equivalence of the relativity and shows its purest, mathematical form. So it's not a placeholder, but rather a metaphor or an example of the guideline. Correct me if I'm wrong @florent-leborgne.

    - Direct
    - Straight to the point
    - Neutral

    When using this tone:
    - Use a minimum of words.
    - Write for scanning and help users locate what matters to them quickly.
    - Use action verbs to 
        - label selection controls
        - describe what settings do or what to do with a setting

>     - Use clear nouns for

Reviewer 2: ```suggestion
    - Use clear nouns for menus or form field names. 
```

>         - menus, form field names 

Reviewer 2: Doesn't seem necessary, considering there's only one list item.
```suggestion
```

**Cases**

    The **informational** tone is the **most used** one. It is everywhere in the UI, and sometimes seems so normal that we forget about it. Yet, it is very important! 
    
    We use it for naming and describing things:

>     - Menus, titles, setting names, hint text, placeholders, intros, and descriptions

Reviewer 2: ```suggestion
    - Menus, titles, setting names, hint text, placeholders, introductions, and descriptions
```

    - In-product assistance (tooltips, help popovers, guided setups)
    - Information callouts
    - Confirmation modals
    - Calls to action (buttons and links)
    - Event logs

>     **Documentation**

Reviewer 2: Same comment as [here](https://github.com/elastic/eui/pull/7710/files#diff-cb2c5b7f50711e795650b30d5f2166939431eac5699008911445a812400d9237R68).

    - Reference content
    - Concepts
    - How-tos
    - Troubleshooting

## Empathetic

**Attributes**

    Something might be wrong and requires attention. Let us help you. Be:

    - Reassuring
    - Professional
    - Precise

    When using this tone, you may:
    - Be more action-oriented than usual to offer direct paths to users.

**Cases**

    The **empathetic** tone is used **sparingly** overall and is usually shown in places that draw the user's attention. 
    
    We use it when notifying users of things that are not immediately critical but still require attention, especially in:

    - Warnings and cautions
    - Upgrade and deprecation notices

## Supportive

**Attributes**

    Something is definitely wrong. Here's why, and how to fix it. Be:

    - Serious
    - Concerned
    - Urgent

    When using this tone, you may:
    - Focus on what went wrong. Don't be apologetic.
    - Help users learn how something happened, how to prevent it, how to fix it.
    - Provide clear actions, or clearly state it if there's no action needed.

**Cases**

>     The **supportive** tone is used **sparingly** and usually becomes visible in forms that draw the user's attention, through callouts or notifications. 

Reviewer 2: Breaking this down a bit so it's easier to read and scan.

```suggestion
    The **supportive** tone is used **sparingly**. It usually becomes visible in forms that draw the user's attention, for example, through callouts or notifications. 
```

    
    We use it when the immediate attention of the user is required, especially in:

    - Errors
    - Licensing or billing-related content
    - Vulnerability and remediation content

## Stern

**Attributes**

    We don't advise this action, but you're the master of your destiny. Be:

    - Deterring
    - Heavy with caution
    - Advisory

    When using this tone, you may:
    - Use stronger words that highlight possible negative effects.
    - Format the content in a way that make the risks stand out.

**Cases**

>     The **stern** tone is used **rarely**, in very specific cases, and for very advanced or admin features.

Reviewer 2: ```suggestion
    The **stern** tone is **rarely** used. Reserve it for very specific cases, and for advanced or administrative features.
```

>     We use it when trying to insist on the high risks an action could pose, especially in:

Reviewer 2: ```suggestion
    We use it when insisting on the high risks an action could pose, especially in:
```
